PKGBUILD

```diff
@@ -0,0 +1,20 @@
+# Maintainer: Shreyansh Khajanchi <shreyansh_k@live.com>
+
+pkgname=gnirehtet
+pkgver=2.1
+pkgrel=1
+pkgdesc="Gnirehtet provides reverse tethering for Android"
+arch=('x86_64')
+url="https://github.com/Genymobile/gnirehtet"
+license=('Apache-2.0')
+depends=('android-tools')
+source=("https://github.com/Genymobile/gnirehtet/releases/download/v$pkgver/gnirehtet-rust-linux64-v$pkgver.zip")
```

Comment:
Just to be sure, it will work only from the next release (where apk location will be configured), right?

Reply:
Yeah! On an update, modify the file hashes and the app version.